first_assesment/service/pokemon.go

```diff
@@ -0,0 +1,44 @@
+package service
+
+import (
+	"first/model"
+	"first/repository"
+	"fmt"
+)
+
+type PokemonService struct {
+	pokemonRepository *repository.PokemonRepository
+}
+
+func NewPokemonService() (*PokemonService, error) {
+	pokemonRepository, err := repository.NewPokemonRepository()
+	if err != nil {
+		return nil, err
+	}
+
+	return &PokemonService{
+		pokemonRepository: pokemonRepository,
+	}, nil
+}
+
+func (s *PokemonService) Saludo() {
+	fmt.Println("Hello world")
+}
```

Comment:
I think this was not intended to be present in the PR, right?

Reply:
yes, it was deleted